MODULE.bazel

```diff
@@ -341,6 +341,13 @@ http_file(
     urls = ["https://github.com/jqlang/jq/releases/download/jq-1.5/jq-osx-amd64"],
 )
 
+http_file(
+    name = "jq_macos_arm64",
+    executable = True,
+    integrity = "sha256-C75hnmY+DeLFUL4v4NJA0HZ5nW+KZStw+gSuqKg2Loo=",
+    urls = ["https://github.com/jqlang/jq/releases/download/jq-1.7.1/jq-macos-arm64"],
```

Comment:
Is it fine this isn't version 1.5? What are the differences?

Reply:
I would like to keep some `jq`s at the minimum supported version (1.5) to ensure that we don't start using features available only in more recent versions. But 1.7 is the first minor release with arm64 builds.

Reply:
I see, I guess 1.7 should be backwards compatible, right? And with other platform staying at 1.5, we should not be able to use new features in 1.7.

Reply:
Can we also add for Linux arm64 and Windows arm64?

Reply:
Ah,, I forgot to send my comment earlier... sorry